data/app.fotema.Fotema.desktop.in.in

```diff
@@ -10,3 +10,5 @@ Keywords=Gnome;GTK;Pictures;Photos;Photography;Viewer;
 # Translators: Do NOT translate or transliterate this text (this is an icon file name)!
 Icon=@icon@
 StartupNotify=true
+# Translators: Do NOT translate or transliterate this text (these are enum types)!
+X-Purism-FormFactor=Workstation;Mobile;
```

Comment:
I'm happy to accept this PR, but what does this line enable in Phosh? Is it just how it is displayed in the Phosh app store?

Reply:
Phosh - the default desktop environment on several mobile linux distributions (e.g. PureOs on the Librem5) - has a filter where by default the home screen only shows apps marked as "Mobile". So this line just tells it the app works well on Mobile and it's fine to display it.